src/table.rs

```diff
@@ -118,6 +118,20 @@ impl Table {
         self.header.as_ref()
     }
 
+    /// Returns the number of currently present columns.
+    ///
+    /// ```
+    /// use comfy_table::Table;
+    ///
+    /// let mut table = Table::new();
+    /// table.set_header(vec!["Col 1", "Col 2", "Col 3"]);
+    ///
+    /// assert_eq!(table.col_count(), 3);
+    /// ```
+    pub fn col_count(&self) -> usize {
+        self.columns.len()
+    }
```

Comment:
This is not always accurate though. See `discover_columns`.
Also, we should stick to the naming convention of the table and don't use any abbreviations. -> `column_count`.

Reply:
> This is not always accurate though. See discover_columns.

Ah yes, you are right. We could make sure to call `discover_columns` before returning the length.

> Also, we should stick to the naming convention of the table and don't use any abbreviations. -> column_count.

Yes, agreed. I will change that!

Reply:
Renamed in 044cf03. I also added the `discover_columns` call.